lib/zip.js

```diff
@@ -13,7 +13,14 @@
 'use strict';
 
 const afs = require('./fs'),
-	AdmZip = require('adm-zip');
+	fs = require('fs-extra'),
+	AdmZip = require('adm-zip'),
```

Comment:
<!--
  0 failure: 
  1 warning:  lib/zip.js line 1...
  
  
  DangerID: danger-id-Danger;
  File: lib/zip.js;
  Line: 17;
-->

- :warning: lib/zip.js line 17 – 'AdmZip' is assigned a value but never used. (no-unused-vars)